lib/components/atoms/Popover/tests/Popover.test.js

```diff
@@ -0,0 +1,15 @@
+import { shallow } from 'enzyme';
+// import renderer from 'react-test-renderer';
+
+import Popover from '../index';
+
+describe('<Popover />', () => {
```

Comment:
The test could be a bit better, at least test the code you wrote, like close and open functionality. Or any other new feature you add. Just snapshot is not enough.